docker-assets/ManageIQContainerBuildProcess.md

```diff
@@ -0,0 +1,97 @@
+## Building and running a ManageIQ Container 
+
+The following steps describe how to build a ManageIQ container which also includes private manageiq <span style="color:blue">provider</span> plugin.
+
+1. In ManageIQ core, developer can override gem dependencies in directory bundler.d/local_plugins.rb. This small helper under bundler.d directory points to the developers version of provider plugin gem. For a given provider with private manageiq-<span style="color:blue">*providers*</span>-plugin gem, add the following lines in bundler.d/local_plugins.rb.
+
+	```
+	group :plugin_group_name, :manageiq_default do
+  	  gem 'manageiq-providers-plugin',  :git =>  'https://manageiq/manageiq-providers-plugin.git',  :branch => "master"
+	end
+	```
+
+
+	Developer may configure bundler  with git user login and password, or personal access token to download and install **manageiq-provider-plugin** from git during **bundle install**
+	```
+    bundle config GITHUB__COM username:password USER:XXXXXXXX --global
+    bundle confg
+     Settings are listed in order of priority. The top value will be used.
+     github.com
+     Set for the current user (/root/.bundle/config): "USER:XXXXXXXX"
+   ```
```

Comment:
I'm not sure we need deep details in this section.  We already have this documented in the guides repo, so it might be better to just say something like

```md
1. In ManageIQ core, set up your repository for plugin development.  You can read more on how to do that [here](/docs/guides/developer_setup/plugins.html).
```

Reply:
If you want to add more specific plugin development setup for things ilke this password section, I would recommend putting it in the plugin guide: https://github.com/ManageIQ/guides/blob/master/developer_setup/plugins.md